src/ehr_components/ui/button.tsx

```diff
@@ -0,0 +1,21 @@
+export interface Prop {
+  children: JSX.Element | string
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void
+}
+
+const Button = (props: Prop) => {
+
+  let { onClick, children } = props;
+
+  return (
+    <button
+      type="button"
+      className="bttn-custom"
+      onClick={onClick}
+    >
+      {children}
+    </button>
```

Comment:
I thinking we call this innertext, let me know what you think

Reply:
if you are talking about the `children` prop, this is the react's way to pull in component composition. 

https://reactjs.org/docs/composition-vs-inheritance.html#containment